openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -152,32 +151,25 @@ An End-User typically authorizes the issuance of Credentials with a specific Cre
 ### Credential Formats and Credential Format Profiles
 
 This specification is Credential Format agnostic and allows implementers to leverage specific capabilities of Credential Formats of their choice.
-To this end, extension points to add Credential Format specific parameters in the Credential Issuer metadata, Credential Offer, Authorization Request, Credential Request, and Batch Credential Request are defined.
+To this end, extension points to add Credential Format specific parameters in the Credential Issuer metadata, Credential Offer, Authorization Request, and Credential Request are defined.
 
 Credential Format Profiles for IETF SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc], ISO mDL [@ISO.18013-5], and W3C VCDM [@VC_DATA] are specified in (#format-profiles).
 Other specifications or deployments can define their own Credential Format Profiles using the above-mentioned extension points.
 
 ### Multiple Credential Issuance
 
-This specification allows for the issuance of Verifiable Credentials through two types of endpoints: the Single Credential Endpoint and the Batch Credential Endpoint.
-
-1. Single Credential Endpoint: This endpoint is used when a single Credential is to be issued. The request to this endpoint includes the necessary parameters for the issuance of one Credential. The response from this endpoint will contain one issued Credential.
-2. Batch Credential Endpoint: This endpoint is used when multiple Credentials need to be issued at once. The request to this endpoint includes the necessary parameters for the issuance of multiple Credentials. The response from this endpoint will contain all the issued Credentials.
+This specification allows for the issuance of Verifiable Credentials through the Credential Endpoint.
+A single request message sent to this endpoint requests the issuance one or more Verifiable Credentials.
```

Comment:
```suggestion
This specification enables the issuance of Verifiable Credentials through the Credential Endpoint. A single request message to this endpoint MAY request the issuance of one or more Verifiable Credentials.
```

Reply:
let's not use normative MAYs in a text that is purely meant to be descriptive.